doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -57,3 +102,34 @@ Example:
 </Button>
 ```
 > note: Here `VisibleBoundsPadding.PaddingMask` is used to prevent the content from being placed outside of the user-interactable area on mobile devices.
+
+### Extended Use Cases
+- Rounded Corner
+    ```xml
+    <Flyout Placement="Full">
+        <Flyout.FlyoutPresenterStyle>
+            <Style BasedOn="{StaticResource BottomDrawerFlyoutPresenterStyle}" TargetType="FlyoutPresenter">
+                <Setter Property="CornerRadius" Value="16,16,0,0" />
+            </Style>
+        </Flyout.FlyoutPresenterStyle>
+        <Border toolkit:VisibleBoundsPadding.PaddingMask="All" Padding="16,16,0,0">
+            <!-- flyout content -->
+        </Border>
+    </Flyout>
+    ```
+    > remarks: `Padding` is used on the flyout content to avoid content being clipped.
+- Custom background
+    ```xml
+    <Flyout Placement="Full">
+        <Flyout.FlyoutPresenterStyle>
+            <Style BasedOn="{StaticResource BottomDrawerFlyoutPresenterStyle}" TargetType="FlyoutPresenter">
+                <Setter Property="Background" Value="SkyBlue" />
+            </Style>
+        </Flyout.FlyoutPresenterStyle>
+        <Border toolkit:VisibleBoundsPadding.PaddingMask="All" >
+        <!--<Border toolkit:VisibleBoundsPadding.PaddingMask="All" Background="SkyBlue">-->
```

Comment:
Maybe remove this? Otherwise add a `<!-- DO NOT DO THIS -->` above it or something. I'm worried people are going to see it and not read the "remarks:" below

Reply:
yeah the DO/DONT is a good idea